technical-reports/Performance-Testing.md

```diff
@@ -0,0 +1,105 @@
+# Performance Testing in TEAMMATES
+
+Authors: [Ronak Lakhotia](https://github.com/RonakLakhotia), [Amrut Prabhu](https://github.com/amrut-prabhu) and [Jacob Li PengCheng](https://github.com/jacoblipech)
+
+* [Introduction](#Introduction)
+* [Why do we need Performance Testing?](#Why-do-we-need-Performance-Testing?)
+* [Overview of the Proposed Solution](#Overview-of-Proposed-Solution)
```

Comment:
> #Why-do-we-need-Performance-Testing?)

@RonakLakhotia I think you need to remove the question mark in the link.

> Overview of the Proposed Solution

Make this consistent with the actual header?

Reply:
It does not work if I remove the question mark from the link

>Make this consistent with the actual header?

Done